quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/TimefoldProcessor.java

```diff
@@ -350,11 +352,6 @@ Some solver configs (%s) don't specify a %s class, yet there are multiple availa
                     "Unused classes ([%s]) found with a @%s annotation.".formatted(String.join(", ", unusedSolutionClassList),
                             PlanningSolution.class.getSimpleName()));
         }
-        // Validate the solution classes target types
-        List<AnnotationTarget> targetList = annotationInstanceCollection.stream()
-                .map(AnnotationInstance::target)
-                .toList();
-        assertTargetClasses(targetList, DotNames.PLANNING_SOLUTION);
     }
```

Comment:
Who calls all this removed code now, and where from?

Reply:
These assertions are contained within the `SolutionDescriptor` and `EntityDescriptor`. Line #1018 is responsible for instantiating the solution descriptor and triggering the related assertions.